README.md

```diff
@@ -333,7 +333,7 @@ You have the following option to trigger a configuration reload:
   Note: this needs to include a valid CRUMB and authentication information e.g. username + token of a user with admin
   permissions. Since Jenkins 2.96 CRUMB is not needed for API tokens.
 - via Jenkins CLI
-- via http POST to `JENKINS_URL/configuration-as-code/reloadWebhook`
```

Comment:
Just removing the exists API endpoint might not be a good practice.

Reply:
? this never existed